README.md

```diff
@@ -23,7 +23,38 @@ The Definitions Of Done (DoD) for our project are aligned with the professor's r
 Uses the [Java](https://phoenixnap.com/kb/install-java-windows) [Spring Boot](https://spring.io/projects/spring-boot/) framework and [Postgresql](https://www.postgresql.org/) with [POJA](https://github.com/hei-school/poja-cli).
 
 #### How to use the app ?
-The principle of POJA is that it deploys directly the application So you can check in the Github Actions and search for the deployed app link.
+The principle of __POJA__ is that it deploys directly the application So you can check in the Github Actions and search for the deployed app link.
+
+#### Requirements:
+   - JDK 17 installation
+   - JAVA_HOME configuration
+   - Database creation on PostgreSQL {database}
+   - List of environment variables
+
+#### Linter and formatter:
+The linter used is Checkstyle, configured with Gradle.
+
+#### Usage:
+   - To clean previous builds:
+```shell
+    ./gradlew clean
+```
+
+   - To build:
+```shell
+    ./gradlew assemble
+```
+
+   - To run tests:
+```shell
+    ./gradlew test
+```
+
+   - To run the application:
+Set Environment variables on CLI and:
+```shell
+    ./gradlew bootRun
```

Comment:
put the sentence "Set Environment variables on CLI and then run:" in the shell as comment

   - To run the application:
   
```shell
    #Set Environment variables on CLI and then run:
    ./gradlew bootRun